index.html

```diff
@@ -0,0 +1,256 @@
+                            <li class="li-lang">Ruby on rails</li>
+                            <li class="li-lang">Rspec</li>
+                            <li class="li-lang">css</li>
+                            <li class="li-lang">Capybara</li>
+                            <li class="li-lang">Selenium</li>
+                        </ul>
+                        </div>
+        
+                    </div>
+                    <div class="project2">
+                        <div class="language-icon"><img src="./images/icon-skills (1).png" alt=""></div>
+                        <div class="lang-text">Skills</div>
+                        <ul class="lang-text-list">
+                            <li class="li-lang" >CodeKit</li>
+                            <li class="li-lang">Github</li>
+                            <li class="li-lang">Codepen</li>
+                            <li class="li-lang">Gitlab</li>
+                            <li class="li-lang">Terminal</li>
+                        </ul>
+                        </div>
+        
+                    </div>
+                    </div>
+                
+                </section>
+                </section>
+    
+                <section class="contact" id="contact">  
+            
+                    <section class="form-section">
+                    
+                        <div class="form-description">
+                        <p class="touch">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+                        </div>
+                    
+                        <div class="contact-form">
+                     <form action="https://formspree.io/f/mjvzwlqy" method="post">
+                       
+                        <ul>
+                            
+                            <li >
+                                <label for="name" >
+                                    <input type="text" id="name"  name="user_name" placeholder="Full name" class="placeholdermargin"  maxlength="30" required>
+                                </label>
+                            </li>
+                            <li>
+                                <label for="mail">
+                                    <input type="email" id="mail" name="user_email" placeholder="Email address" class="placeholdermargin" required>
+                                </label>
+        
+                            </li>
+                            <li>
+                                <label for="msg">
+                                    <textarea name="user_message" id="msg" cols="30" rows="5" maxlength="500" placeholder="Enter text here" class="placeholdermargin" required></textarea>
+                                </label>
+                            </li>
+                            <li class="button back-image">
+                                <button type="submit" class="btn">Get in touch</button>
```

Comment:
- [ ] Good job implementing some changes requested by previous reviewers but I am sorry to say that there are some issues still there that need to be fixed in order to get ahead. In order to fix the requested changes, there were some new unwanted changes got introduced.for example: 
- [ ] what I can see is that the whole form section is not centered right now. The heading of the form section is not centered. there needs to be equal margin on both sides as in the figma design. 
- [ ] Kindly consider centering the `Get in touch` button with the form section.
- [ ] the social media icons are not aligned properly, the last one has extra left margin. kindly consider giving them equal margin and centering them as in the figma design because we need to stick with the figma design.
- [ ] The placeholder text in the `textarea` doesn't have proper margin as in the figma design. kindly consider fixing it as mentioned by the previous reviewer.
- [OPTIONAL] Kindly consider adding your `email` address under the social media icons as in the figma design.
- [OPTIONAL] there are some animation in your hero section which is continuously moving and then causing the form section to move up and down, which I think is not a pleasant user experience. kindly consider removing that animation or placing a setTimeout on that.

your design
![asd](https://user-images.githubusercontent.com/96683843/213763771-353c94ee-4d6b-43a8-a588-eff2c0a575d8.jpg)

figma design
![qwe](https://user-images.githubusercontent.com/96683843/213763583-ab3dc244-5fae-4d0b-b322-c51dbd48dc4b.jpg)